sys/kern/sched.c

```diff
@@ -211,6 +211,16 @@ void sched_maybe_preempt(void) {
   }
 }
 
+void sched_yield(void) {
```

Comment:
In kernel space we can just call it `yield` as they do: http://bxr.su/NetBSD/sys/kern/kern_synch.c#272